README.md

```diff
@@ -246,8 +245,7 @@ Sample result:
 ### Listing buckets
 
 ```js
-client.listBuckets([options])
-  .then(({data}) => ...);
+const { data } = await client.listBuckets([options]);
```

Comment:
You wrote it `{data}` (without the spaces) through the rest of the file, not sure what Prettier would want.